scripts/aws-cni-support.sh

```diff
@@ -17,57 +17,59 @@
 # Set language to C to make sorting consistent among different environments.
 export LANG=C
 
-set -e
+set -euf -o pipefail
 LOG_DIR="/var/log/aws-routed-eni"
+[[ ! -d ${LOG_DIR} ]] && mkdir -p ${LOG_DIR}
 
 # collecting L-IPAMD introspection data
-curl http://localhost:61678/v1/enis         > ${LOG_DIR}/eni.output
-curl http://localhost:61678/v1/pods         > ${LOG_DIR}/pod.output
-curl http://localhost:61678/v1/networkutils-env-settings > ${LOG_DIR}/networkutils-env.output
-curl http://localhost:61678/v1/ipamd-env-settings > ${LOG_DIR}/ipamd-env.output
-curl http://localhost:61678/v1/eni-configs  > ${LOG_DIR}/eni-configs.output
+curl http://localhost:61678/v1/enis         > ${LOG_DIR}/eni.out
+curl http://localhost:61678/v1/pods         > ${LOG_DIR}/pod.out
+curl http://localhost:61678/v1/networkutils-env-settings > ${LOG_DIR}/networkutils-env.out
+curl http://localhost:61678/v1/ipamd-env-settings > ${LOG_DIR}/ipamd-env.out
+curl http://localhost:61678/v1/eni-configs  > ${LOG_DIR}/eni-configs.out
 
 # metrics TODO not able to use LOG_DIR
-curl http://localhost:61678/metrics 2>&1 > /var/log/aws-routed-eni/metrics.output
+curl http://localhost:61678/metrics 2>&1 > /var/log/aws-routed-eni/metrics.out
```

Comment:
There's no reason here for not using `${LOG_DIR}` as well, or if there is then that reason should be written down more clearly.